### PR TITLE
Fixes for flyte-core helm chart to allow deploying cluster resource sync as a standalone process

### PR DESCRIPTION
**Pull request by katrogan**
- Update flyteadmin image version to latest with standalone cluster resource controller code changes
- Use boolean switch in the cluster resource config to set standalone mode

### CODE REVIEW

File: charts/flyte-core/values.yaml [hunk 1]
```diff
@@ -638,15 +638,17 @@ external_events:
 cluster_resource_manager:
   # -- Enables the Cluster resource manager component
   enabled: true
-  # -- Starts the cluster resource manager in standalone mode with requisite auth credentials to call flyteadmin service endpoints
-  standalone_deploy: false
```
Comment (EngHabu) [inline charts/flyte-core/values.yaml:642]: Wouldn't moving this break people's existing values files, if any? is there a reason to move it?

Comment (katrogan) [reply charts/flyte-core/values.yaml:642]: it's already part of the config, see: https://github.com/flyteorg/flyteadmin/blob/30862d414125eda4ef74455e4614100b522d2af8/pkg/runtime/interfaces/cluster_resource_configuration.go#L46

Comment (katrogan) [reply charts/flyte-core/values.yaml:642]: and this has only existed in a beta release so far

**Review by EngHabu — DISMISSED** [review]

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ on:
     branches:
```
[the remark this answers is not in the document]
Comment (katrogan) [reply .github/workflows/tests.yml:4]: I am not sure... @evalsocket is this necessary?

[the remark this answers is not in the document]
Comment (katrogan) [reply .github/workflows/tests.yml]: removed.

**Comment by katrogan** [comment]
PTAL @EngHabu

**Review by EngHabu — APPROVED** [review]

**Pull request merged**